geode-web-api/src/main/java/org/apache/geode/rest/internal/web/controllers/AbstractBaseController.java

```diff
@@ -28,15 +28,16 @@
 import java.util.Set;
 import java.util.concurrent.atomic.AtomicLong;
 
+import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.JsonToken;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.logging.log4j.Logger;
-import org.json.JSONArray;
```

Comment:
I believe these imports come from `geode-json` project. Is this on the way to getting rid of that proj?

Reply:
Yes - the aim is to eventually remove `geode-json`